chips/lowrisc/src/gpio.rs

```diff
@@ -104,7 +111,7 @@ impl GpioPin {
     }
 
     pub fn handle_interrupt(&self) {
-        let regs = self.registers;
+        let regs = self.gpio_registers;
```

Comment:
Is there a reason for this renaming (and I mean the local `regs` variable from `self.gpio_registers`; the `registers` -> `gpio_registers` makes sense)?

I see how this is sort of copying what exists elsewhere in Tock, but I think that's more an accident of legacy MMIO design than intentional, and is hopefully on the way out (see #1973). I think while this is renaming this should just remove the `regs`.

Reply:
Removed